applications_tests/lethe-fluid/gls_vof_2_isothermal_compressible_fluids.output

```diff
@@ -28,4 +28,4 @@ Transient iteration: 2        Time: 0.2      Time step: 0.1      CFL: 1.53178
 VOF Mass Conservation
 ----------------------
     time     surface_fluid_0 mass_per_length_fluid_0 momentum-x_fluid_0 momentum-y_fluid_0 surface_fluid_1 mass_per_length_fluid_1 momentum-x_fluid_1 momentum-y_fluid_1 sharpening_threshold 
-2.000000e-01    6.579652e-01            7.895557e-01      -2.004270e-02       4.448404e-02    3.420348e-01            1.710176e+01       8.361416e-01      -1.863500e+00         5.000000e-01 
+2.000000e-01    6.579587e-01            7.895478e-01      -2.005946e-02       4.446851e-02    3.420413e-01            1.710209e+01       8.365478e-01      -1.862997e+00         5.000000e-01
```

Comment:
nor this one! The rest of the application tests returned to their previous values (before the square)

Reply:
same here :)